events/management/commands/populate_local_event_cache.py

```diff
@@ -30,6 +32,26 @@ def handle(self, *args, **options):
             event_dict[i[0]].update(i[1:])
             event_dict[i[0]].discard(None)
 
-        event_strings = {k: " ".join(v) for k, v in event_dict.items()}
+        event_strings = {k: " ".join(v).replace('\n', ' ').replace('\r', ' ') for k, v in event_dict.items()}
+
+        cache.set('local_ids', event_strings)
+
+        inet_events = Event.objects.filter(location__id__endswith='internet',
+                                           end_time__gte=datetime.utcnow().replace(tzinfo=pytz.utc),
+                                           deleted=False,
+                                           ).values_list('id', 'name', 'description', 'short_description',
+                                                         'name_en', 'description_en', 'short_description_en',
+                                                         'name_sv', 'description_sv', 'short_description_sv',
+                                                         'keywords__name_fi', 'keywords__name_sv', 'keywords__name_en',  # noqa E501
+                                                         'location__street_address_fi', 'location__street_address_sv',
+                                                         'location__name_fi', 'location__name_sv', 'location__name_en',  # noqa E501
+                                                         'location__description_fi', 'location__description_sv',
+                                                         'location__description_en')
+        event_dict = {i[0]: set() for i in inet_events}
+        for i in inet_events:
+            event_dict[i[0]].update(i[1:])
+            event_dict[i[0]].discard(None)
+
+        event_strings = {k: " ".join(v).replace('\n', ' ').replace('\r', ' ') for k, v in event_dict.items()}
```

Comment:
This event_strings creation could be in a reusable function, because the pattern is always the same.